packages/ti_eclecticiq/manifest.yml

```
@@ -0,0 +1,42 @@
+format_version: 3.0.0
+name: ti_eclecticiq
+title: EclecticIQ Outgoing Feeds Integration
+version: 0.1.0
+description: Download data from EclecticIQ IC outgoing feeds
+type: integration
+categories:
+  - security
+  - threat_intel
+conditions:
+  kibana:
+    version: ^8.8.2
```

```suggestion
    version: ^8.10.3
```
To use transform's alias